src/inspector/node_protocol.pdl

```diff
@@ -98,6 +98,81 @@ experimental domain NodeWorker
       SessionID sessionId
       string message
 
+experimental domain NodeNetwork
+  depends on Network
+
+  # Fired when page is about to send HTTP request.
+  event requestWillBeSent
+    parameters
+      # Request identifier.
+      Network.RequestId requestId
+      # Request data.
+      Network.Request request
+      # Timestamp.
+      Network.MonotonicTime timestamp
+      # Timestamp.
+      Network.TimeSinceEpoch wallTime
+
+  # Fired when HTTP response is available.
+  event responseReceived
```

Comment:
Generally, inspector domain events should be delivered to client only if the client enables the domain by `<domain>.enable` command, like [`Network.enable`](https://chromedevtools.github.io/devtools-protocol/tot/Network/#method-enable).

Reply:
Could you please take a look at e53ebec501fdc6adfbb3cb0b911664862ab64892?

Reply:
Yeah, however in this test `test/parallel/test-inspector-emit-protocol-event.mjs` there is no `NodeNetwork.enable` required to receive network events.

Reply:
Ah I see, let me fix it.

Reply:
https://github.com/nodejs/node/pull/53593/commits/6a80de660290aeb703eca2398d7f7a4bd3ed1cff